mgmtfn/dockplugin/netDriver.go

```diff
@@ -755,21 +772,14 @@ func deleteNetworkHelper(networkID string) error {
 
 		err = cluster.MasterDelReq(url)
 		if err != nil {
-			log.Errorf("Failed to delete network: %s", err.Error())
-			return errors.New("Failed to delete network")
+			msg := fmt.Sprintf("Failed to delete network: %s", err.Error())
+			log.Errorf(msg)
+			return errors.New(msg)
 		}
 		log.Infof("Deleted contiv network %v", networkID)
 	} else {
 		log.Infof("Could not find contiv network %v", networkID)
```

Comment:
We should still be bubbling up an error here (like bad request to user for deleting a network that didn't exist), but happy to see this as a new ticket

Reply:
At the moment, I don't think return error here will bubbling up to the docker network create command. We have to have another ticket to investigate it.